pythonforandroid/bootstraps/webview/__init__.py

```diff
@@ -21,31 +21,28 @@ def assemble_distribution(self):
             with open('local.properties', 'w') as fileh:
                 fileh.write('sdk.dir={}'.format(self.ctx.sdk_dir))
 
-        arch = self.ctx.archs[0]
-        if len(self.ctx.archs) > 1:
-            raise ValueError('built for more than one arch, but bootstrap cannot handle that yet')
-        info('Bootstrap running with arch {}'.format(arch))
-
         with current_directory(self.dist_dir):
             info('Copying python distribution')
 
-            self.distribute_libs(arch, [self.ctx.get_libs_dir(arch.arch)])
-            self.distribute_aars(arch)
             self.distribute_javaclasses(self.ctx.javaclass_dir,
                                         dest_dir=join("src", "main", "java"))
 
-            python_bundle_dir = join(f'_python_bundle__{arch.arch}', '_python_bundle')
-            ensure_dir(python_bundle_dir)
-            site_packages_dir = self.ctx.python_recipe.create_python_bundle(
-                join(self.dist_dir, python_bundle_dir), arch)
+            for arch in self.ctx.archs:
+                self.distribute_libs(arch, [self.ctx.get_libs_dir(arch.arch)])
+                self.distribute_aars(arch)
+
+                python_bundle_dir = join(f'_python_bundle__{arch.arch}', '_python_bundle')
+                ensure_dir(python_bundle_dir)
```

Comment:
The sdl2 bootstrap has these lines before lines 31 and 32. But this doesn't affect anything at all.

Reply:
Sure, I can make it match the order there.